dev/ci/ci-vst.sh

```diff
@@ -11,9 +11,9 @@ if [ "$DOWNLOAD_ONLY" ]; then exit 0; fi
 
 export COMPCERT=bundled
 
-ulimit -s
-ulimit -s 65536
-ulimit -s
+sudo ulimit -s
+sudo ulimit -s 65536
+sudo ulimit -s
 ( cd "${CI_BUILD_DIR}/vst"
   make IGNORECOQVERSION=true
 )
```

Comment:
These changes should not be part of this PR. Move them to another PR if you want them to be discussed.

Reply:
The current PR uselessly includes PR #17902 which itself mistakenly included this debugging code. Now fixed.